css/style.css

```diff
@@ -0,0 +1,171 @@
+#container {
+  width: 100%;
+  overscroll-behavior: none;
+}
+
+#header-img {
+  width: 300px;
+  height: 40px;
+  margin: 10px 0 0 20px;
+}
+
+#header {
+  width: 100%;
+  position: fixed;
+  top: 0;
+  display: flex;
+  overflow: hidden;
+  background-color: #e8e58e;
+}
+
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+#nav-bar {
+  margin-right: 50px;
+}
+
+#nav-bar ul li {
+  color: black;
+  list-style-type: none;
+  float: left;
+  padding: 0 10px;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+  font-size: 20px;
+}
+
+#form-section {
+  width: 50%;
+  margin-top: 5%;
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  padding: 10px;
+}
+
+.desc > h2 {
+  color: #7a2ff4;
+  font-size: 30px;
+}
+
+#form-section h2 {
+  text-align: center;
+}
+
+#form-section #form {
+  text-align: center;
+}
+
+#email {
+  width: 50%;
+  height: 40px;
+  font-size: 18px;
+  padding: 5px;
+}
+
+#submit {
+  width: 20%;
+  height: 40px;
+  margin-top: 8px;
+  color: #fff;
+  font-size: 18px;
+  font-weight: bold;
+  background-color: #30d3e5;
+  border: 0;
+}
+
+#how-it-works {
+  margin: 80px 400px;
+}
+
+#video {
+  width: 1000px;
+  height: 500px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+@media (max-width: 650px) {
+  nav > ul {
+    flex-direction: column;
+  }
+}
+
+#features {
+  background-color: #f1f1f1;
```

Comment:
You did a good job in writing this CSS for your project.
There are a few areas where you repeat the same lines of code.
Please try as much as possible to use the DRY method.